experimental/packages/sdk-logs/src/LogRecord.ts

```diff
@@ -140,6 +140,9 @@ export class LogRecord implements ReadableLogRecord {
         this._logRecordLimits.attributeCountLimit &&
       !Object.prototype.hasOwnProperty.call(this.attributes, key)
     ) {
+      if (this.droppedAttributesCount === 1) {
```

Comment:
Adding a comment here mentioning this logic is to prevent excessive logging and only creating a single warning per LogRecord would be great